ban/core/models.py

```diff
@@ -64,6 +60,9 @@ class Municipality(NamedModel):
     insee = db.CharField(max_length=5, unique=True)
     siren = db.CharField(max_length=9, unique=True, null=True)
 
+    class Meta:
+        order_by = ['insee']
```

Comment:
I'd move them as class attribute of the Resource classes, and use it when building the Resource collection. The idea is not to have a default `order_by` after all ;)